src/AppBundle/Resources/views/themes/app/page_path.html.twig

```diff
@@ -1,13 +1,17 @@
 {% if show_path and path_array|length > 1 %}
-    <div class="breadcrumb-wrapper">
-        <div class="container">
-            <ol class="breadcrumb">
-                {% for path_item in path_array %}
-                    {% if path_item.url != false %}
-                        <li><a href="{{ path_item.url }}">{{ path_item.text }}</a></li>
-                    {% endif %}
-                {% endfor %}
-            </ol>
-        </div>
-    </div>
+    <nav class="breadcrumb-wrapper" aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            {% for path_item in path_array %}
+                {% if path_item.url != false and path_item.location.contentInfo.contentTypeIdentifier not in ezpublish.configResolver.parameter('container_content_types', 'ngsite') %}
```

Comment:
What was the logic behind this? In theory, if a content is not a container, then it should not have subitems below it, meaning, that every item in the path, by extension, is a container. That means that `container_content_types` check can be ommited.

@Ljudevit WDYT?

Reply:
@emodric the idea behind this was to create a breadcrumbs list only with items that should have the full view. 

Containers should be ignored in building a page path and only non-container (well, not ignored ones in configuration) should be displayed.

Example: 

frontpage -> container -> news structure should only have frontpage and news items in breadcrumbs.

Reply:
Ok, makes sense. Then `ezpublish.configResolver.parameter('container_content_types', 'ngsite')` just needs to be taken out in a variable and this is good to go :+1:

Reply:
PR updated, ezpublish.configResolver.parameter('container_content_types', 'ngsite') removed